code/Modules/Input/Input.cc

```diff
@@ -34,6 +34,14 @@ Input::IsValid() {
 }
 
 //------------------------------------------------------------------------------
+void
+Input::Update() {

```

Comment:
With the new local emscInputMgr::updateGamepads() PostRunLoop callback we don't need this global per-frame Update() method in the Input system.